torchvision/transforms/transforms.py

```diff
@@ -321,7 +321,14 @@ def __repr__(self):
 
 
 class RandomHorizontalFlip(object):
-    """Horizontally flip the given PIL Image randomly with a probability of 0.5."""
+    """Horizontally flip the given PIL Image randomly with a probability.
+
+    Args:
+        p (float): probability of image is flipped. Default value is 0.5
```

Comment:
maybe we could say instead `probability of the image being flipped. Default value is 0.5`